usaspending_api/spending_explorer/v2/filters/type_filter.py

```diff
@@ -183,9 +180,21 @@ def type_filter(_type, filters, limit=None):
                     award["name"] = code
             actual_total += award["total"]
 
-        alt_set = alt_set[:limit] if _type == "award" else alt_set
+        result_set = list(alt_set)
+
+        # we need to get the File B data for the same set of filters, so we re-run the spending_filter but without setting the _type to any of the alt keys.
+        alt_set2, queryset2 = spending_filter(alt_set, queryset, filters, "")
+        expected_total = queryset2.aggregate(total=Sum("amount"))["total"]
+        unreported_obj = {"id": None, "code": None, "type": _type, "name": UNREPORTED_DATA_NAME, "amount": None}
```

Comment:
If I understand the ticket, this is separate from unreported data since it is reported in File B. This is non-award spending. It would be best to rename the variables to avoid confusion